### ISSUE TITLE
ExpoKit Android Project Doesn't initialise at first install

### ISSUE THREAD

**Issue by gitsuraj** [expo/expo#2100]
# READ THIS FIRST, PLEASE!

The App Store version of the Expo iOS client recently had to remove the QR code scanner from the app. If you intend to load your app on a physical iOS device, please use `exp send` with exp, the “Send” button with XDE, or the "s" hotkey with create-react-native-app to send your project development URL to your phone by SMS or email.

Read more at https://blog.expo.io/upcoming-limitations-to-ios-expo-client-8076d01aee1a. We apologize for any inconvenience this may cause you!

### Attention

  If you are unable to create a reproducible Snack example that demonstrates the bug/issue,
  please first post in our forums at https://forums.expo.io/ before opening an issue here! 
  (Unless the bug/issue pertains to a standalone, non-detached build)
  
### Environment
OS : MacOS
node : v8.11.3
react-native: 0.55.2
"expo": 27.1.0,

Expokit, Android,Standalone
### Steps to Reproduce

(Write your steps here:)

1. Publish your javascript bundle exp publish
2. Build apk form Android Studio
3. Install apk in Android device

### Expected Behavior

After installation application should run as expected .

(Write what you thought would happen.)

### Actual Behavior
After splash screen there will be loading screen and application doesn't start. If I close application and open it again it will start as expected.

(Write what happened. Add screenshots!)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

**Comment by yuyic** [comment]
I have the same issue, If wifi changed, it will start as expected.

**Comment by sjchmiela** [comment]
Hey! Are you building an APK for development, or for release?

If you're building an APK for release, note that for the JS bundle to be included in the APK at least one version of your experience must have been published. Then, generate a signed APK again and try opening the app.

If you're building an APK for debug, note that before building it you'll first need to start the packager with `expo start` for the APK to work. 🙂

**Comment by gitsuraj** [comment]
@sjchmiela Yes I did published my bundle on expo and build signed APK but my issue is if there is no internet App wont't work in first installation. If I connect to internet and load again it will work as expected as experienced by @yuyic. I am expecting it to work offline without internet as when I publish it did create bundle files for javascript for me.
